test/conftest.py

```diff
@@ -11,13 +11,16 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption('--mode', choices=ALL_MODES, action="append", dest="modes",
+    parser.addoption('--mode', choices=ALL_MODES, action="append", dest="modes", required=True,
```

Comment:
No. "--mode" is NOT required for running test/cqlpy and test/alternator. This will break running test/cqlpy and test/alternator via either with test/alternator/run or against an already running Scylla.

Reply:
done